app/src/components/dialog/FailureDialog.tsx

```diff
@@ -0,0 +1,40 @@
+import { DialogContentText } from '@material-ui/core';
+import ComponentDialog from 'components/dialog/ComponentDialog';
+import React from 'react';
+
+export interface IPublishDialogs {
+  failureTitle: string;
+  failureMessage: string;
+  failureBody?: string;
+  open: boolean;
+  setOpen: (isEmpty: boolean) => void;
```

Comment:
Nit: this should be called something like `onClose` because that is event that will trigger a call to whatever function the parent assigns to this prop. The parent can then decide if it closes the dialog or not.